src/pages/news/news-page/index.js

```diff
@@ -0,0 +1,3 @@
+import NewsPage from './news';
```

Comment:
Name of the component should match with the name of file and folder

Reply:
Redone!